restcomm/restcomm.sms/src/main/java/org/restcomm/connect/sms/SmsService.java

```diff
@@ -208,8 +218,8 @@ private void message(final Object message) throws IOException {
 
                 final SipServletResponse trying = request.createResponse(SipServletResponse.SC_TRYING);
                 trying.send();
-
-                ActorRef session = session();
+                //TODO:do extensions check here too?
```

Comment:
@abdulazizali77 yes we need extension check here also. Those requests are MESSAGE requests from registered clients that Restcomm will proxy out to.